nuxt.config.js

```diff
@@ -92,6 +92,9 @@ export default {
   styleResources: {
     scss: ['./assets/styles/_colors.scss'],
   },
+  storybook: {
+    stories: ['~/udb3-ui/**/*.stories.js'],
```

Comment:
If we prefix the ui library name, component names, etc with `udb3`, how hard would it be to rename those later if we ever decide to use this in other projects and it should become more generic? I do understand the usefulness of a prefix though. Just trying to estimate if it would be okay to go with something as specific as `udb3` right now or if we should think for a moment about a more generic prefix. But that will also be hard. Do you think it would be doable to rename this later if we have 100+ components? I genuinely don't know.

Reply:
(One specific place we might want to re-use these is projectaanvraag for example, where external parties register projects and get an API key etc.)

Reply:
just `Button` or `button` wont be possible. But we could use `s-button` for storybook-button for example. It's shorter and similar to b-button from bootstrap

Reply:
and for the library we could also go for something more general `s-ui` or `sb-ui` or `sb-components` are options or nothing to do with storybook, maybe `custom-components` and then `c-button` `c-modal` and so on?